Cargo.toml

```diff
@@ -546,6 +546,10 @@ once_cell = "1.10.0"
 ordered-float = "3.9.1"
 ouroboros = "0.15.6"
 owo-colors = "3.5.0"
+# TODO: Do we actually use serde?
+p256 = { version = "0.13.2" } #, features = ["serde"] }
```

Comment:
According to https://docs.rs/p256/latest/p256 ...
```
The elliptic curve arithmetic contained in this crate has never been independently audited!
```

Reply:
taking the risk?

Reply:
We also tried the [openssl crate](https://crates.io/crates/openssl) for p256 but it was nearly impossible to figure out where their rng was coming from in their underlying C code. Generally the `RustCrypto` libraries seem to be well-regarded in the blockchain cryptography space. We should still discuss whether we want to have an audit done ourselves on their code

Reply:
Discussed this further with @gedigi and decided an audit on `RustCrypto` isn't necessary for now